components/director/internal/domain/fetchrequest/repository.go

```diff
@@ -0,0 +1,76 @@
+package fetchrequest
+
+import (
+	"context"
+
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+	"github.com/kyma-incubator/compass/components/director/internal/repo"
+	"github.com/pkg/errors"
+)
+
+const fetchRequestTable string = `public.fetch_requests`
+
+var tableColumns = []string{"id", "tenant_id", "api_def_id", "event_api_def_id", "document_id", "url", "auth", "mode", "filter", "status_condition", "status_timestamp"}
```

Comment:
`fetchRequestColumns`? Acoording to `runtime` repository impl.?

Reply:
I want it more generic, `tableColumns` sounds better for me